apps/examples/testcase/ta_tc/task_manager/utc/utc_taskmanager_main.c

```diff
@@ -479,6 +479,20 @@ static void utc_taskmanager_unicast_p(void)
 	TC_SUCCESS_RESULT();
 }
 
+static void utc_taskmanager_broadcast_empty(void)
```

Comment:
It looks a part of `utc_taskmanager_broadcast_n`.

Reply:
I agree that opinion.
I Move this code into utc_taskmanager_broadcast_n.